doc/python/box-plots.md

```diff
@@ -71,6 +71,49 @@ fig = px.box(df, x="time", y="total_bill", points="all")
 fig.show()
 ```
 
+### Choosing The Algorithm For Computing Quartiles
+
+By default, quartiles for box plots are computed using a linear algorithm method (see #10 listed on [http://www.amstat.org/publications/jse/v14n3/langford.html](http://www.amstat.org/publications/jse/v14n3/langford.html) for more details). However, you can also choose to use an `exclusive` or an `inclusive` algorithm to compute quartiles. 
```

Comment:
By `linear algorithm method`, did you mean a method using linear interpolation between consecutive values? I'm not sure I understand.

Reply:
I myself am not quite sure what makes the default algorithm `linear` in this context. Perhaps your suggestion is correct?

However, as `linear` is the attribute name used to refer to default algorithm, I felt that it was safe to refer to it as a `linear` algorithm in these docs.

Reply:
ok, so how about "using the `linear` method (for linear interpolation, see #10...)"?

Reply:
How about this?

```
By default, quartiles for box plots are computed using the `linear` method (for more about linear interpolation, see #10 listed on [http://www.amstat.org/publications/jse/v14n3/langford.html](http://www.amstat.org/publications/jse/v14n3/langford.html) and [https://en.wikipedia.org/wiki/Quartile](https://en.wikipedia.org/wiki/Quartile) for more details). 
```